src/main/java/jp/ikedam/jenkins/plugins/updatesitesmanager/ManagedUpdateSite.java

```diff
@@ -130,24 +127,6 @@ public String getNote()
     }
     
     /**
-     * Returns the path to the CA certificate file.
-     * 
-     * The file to store the CA certificate temporary.
-     * 
-     * @return the path to the CA certificate file
-     */
-    protected File getCaCertificateFile()
-    {
-        File caCertificateDir = new File(Jenkins.getInstance().getRootDir(), "update-center-rootCAs");
-        if(!caCertificateDir.exists())
-        {
-            assert(caCertificateDir.mkdir());
-        }
-        
-        return new File(caCertificateDir, String.format("tmp-ManageUpdateSite-%s.crt", getId()));
-    }

```

Comment:
It's ok though this breaks the binary compatibility.
I don't think other codes accessing this method.
(actually, I'm not sure why I had this method `protected`)